tests/core/channels/test_twilio_voice.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_twilio_voice_twiml_response_text():
+
+    from rasa.core.channels.twilio_voice import TwilioVoiceInput
+    from rasa.core.channels.twilio_voice import TwilioVoiceCollectingOutputChannel
+
+    tv = TwilioVoiceInput("woman")
+
+    output_channel = TwilioVoiceCollectingOutputChannel()
+
+    await output_channel.send_text_message(recipient_id="Chuck Norris", text="Test:")
+    assert len(output_channel.messages) == 1
+    assert output_channel.messages[0]["text"] == "Test:"
+
+    twiml = tv.build_twilio_voice_response(output_channel.messages)
+    assert (
+        str(twiml)
+        == '<?xml version="1.0" encoding="UTF-8"?><Response><Gather action="/webhooks/twilio_voice/webhook" actionOnEmptyResult="true" input="speech" speechTimeout="auto"><Say voice="woman">Test:</Say></Gather></Response>'
+    )
+
+
+@pytest.mark.asyncio
+async def test_twilio_voice_twiml_response_buttons():
+
+    from rasa.core.channels.twilio_voice import TwilioVoiceInput
```

Comment:
can we please use global imports? In my opinion there is no advantage of using local ones in this case